app/workers/issue_assigner_worker.rb

```diff
@@ -18,13 +23,23 @@ def work(trigger_id, settings = {})
 
       # Filter our bot
       collaborators = collaborators.select { |c| c.login != ENV.fetch('GITHUB_BOT_LOGIN') }
+>>>>>>> master
 
       # Choose one randomly
       collaborator = collaborators.sample
       assignee = collaborator.login
 
       # Assign Issue
       client.update_issue(repo_full_name, payload['issue']['number'], :assignee => assignee)
+
+      Task.create!({
+        behavior: "issue_assigner",
+        github_id: payload['issue']['number'],
+        trigger_id: trigger_id,
+        repo_id: Repo.find_by!({
```

Comment:
you should be able to use `trigger.repo` here. I think this would be better at the beginning of the method so that if it fails, it does so before api requests are made?